doc/nrf/ug_gzll.rst

```diff
@@ -7,22 +7,15 @@ Gazell Link Layer
    :local:
    :depth: 2
 
-.. gzll_intro_start
-
-Gazell is a protocol for setting up a robust wireless link between a single Host and up to eight Devices in a star network topology.
-It is designed to minimize power consumption in power-sensitive wireless desktop products and is also suitable for a range of other wireless applications.
-
-.. gzll_intro_end
-
-To minimize the power consumption of the power-sensitive peripheral devices, Gazell uses the central hub (host side) with its more relaxed power constraints to keep the link open while the peripheral devices can sleep and save on power consumption.
+To minimize the power consumption of the power-sensitive peripheral devices, Gazell uses the central hub (Host side) with its more relaxed power constraints to keep the link open while the peripheral devices can sleep and save on power consumption.
```

Comment:
Would it make sense to make this part more definition-like?
`Gazell Link Layer minimizes the power consumption of the power-sensitive peripheral devices. Gazell uses the central hub (Host side)...`
I mean, to introduce what the Link Layer is at the start of the page...

Reply:
Yes, I agree and I have implemented your suggestion.
Let me know if anything else should be done for this part.